src/daemon/plugin.h

```diff
@@ -173,6 +173,7 @@ typedef struct user_data_s user_data_t;
 
 struct plugin_ctx_s {
   char *name;
+  double align_read;
```

Comment:
I'm still thinking if `align_read` should be of `cdtime_t` type....

Reply:
Done.